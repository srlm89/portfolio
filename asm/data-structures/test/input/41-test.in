ERASE actual/41-test.out
INSERT 14 6.813328 Pursuit_Of_Happyness[2006]   720p
PRINT_IN_ORDER actual/41-test.out
INSERT 2 9.678840 Eragon [2006] [720p]
INSERT 7 1.721304 Jumanji [720p]
REMOVE_ID 14
PRINT_IN_ORDER actual/41-test.out
INSERT 28 4.922683 American Pie Book Of Love   720p
INSERT 3 7.457088 Angels & Demons [2009] [1080p]
INSERT 17 0.616942 X-Men 2   1080p
REMOVE_ID 7
PRINT_IN_ORDER actual/41-test.out
INSERT 40 3.442556 Braveheart 720p
INSERT 9 5.147998 The Mist   720p
INSERT 41 4.001350 Oceans.Eleven.2001.720p.HDDVD.x264-SiNNERS
INSERT 10 7.749133 Lucky.Number.Slevin.2006.720p.HDDVD.x264-ESiR
REMOVE_ID 28
REMOVE_ID 10
PRINT_IN_ORDER actual/41-test.out
INSERT 27 2.355438 Watchmen [720p]
INSERT 25 9.097009 The Lion King [1994] [720p]
INSERT 18 4.257201 Demo (Sony, Sharp, THX and more)
INSERT 49 9.497091 CJ7   720p
INSERT 46 9.913062 Oceans.Thirteen.2007.720p.HDDVD.x264-SiNNERS
REMOVE_ID 3
REMOVE_ID 2
PRINT_IN_ORDER actual/41-test.out
INSERT 21 7.419324 contents.txt
INSERT 34 3.202666 Core   720p
INSERT 36 0.530856 Rocky 2 [1979] [BluRay] [720p] [AC3]
INSERT 6 5.319696 Pixar.Short.Films.Collection.Volume.1.2007.m-HD.x264-TlN50p
INSERT 48 2.831868 Jeniffer's Body ( 2009)    1080p
INSERT 47 0.533557 Deep Sea
REMOVE_ID 25
REMOVE_ID 18
REMOVE_ID 41
PRINT_IN_ORDER actual/41-test.out
INSERT 26 3.668788 Alone in the dark   720p
INSERT 50 5.169793 Braveheart 720p
INSERT 62 5.557410 Lord Of The Ring 2 - The Two Towers (ExtEd) [1080p]
INSERT 4 5.939027 The Brave One 720P
INSERT 35 9.163826 Alvin And The Chipmunks [720p]
INSERT 11 6.491815 Harold & Kumar   720p
INSERT 45 6.077245 Blood - The Last Vampire [2009] [720p]
REMOVE_ID 4
REMOVE_ID 34
REMOVE_ID 36
PRINT_IN_ORDER actual/41-test.out
INSERT 13 7.774472 Liar Liar    720p
INSERT 15 7.544752 Final Fantasy The Spirits Within    720p
INSERT 30 5.112269 contents.txt
INSERT 55 7.714682 Meet The Parents    720p
INSERT 61 7.066163 Death Race   720p
INSERT 70 8.577854 James Bond 09 - The Man with the Golden Gun [1974]
INSERT 22 9.252123 City of Embers    720p
INSERT 39 0.057899 Lust Caution   720p
REMOVE_ID 35
REMOVE_ID 61
REMOVE_ID 17
REMOVE_ID 48
PRINT_IN_ORDER actual/41-test.out
INSERT 66 2.989426 Etenal Sunshine of the Spotless Mind [2004] [Bluray] [1080p] [DTS] [EN]
INSERT 63 1.182145 The.X.Files.Fight.The.Future.1998.720p.BluRay.x264-SiNNERS
INSERT 8 8.228329 Superman Batman Public Enemies 2009 720p
INSERT 42 3.715998 Ice Age 3 Dawn of the Dinosaurs [2009] [720p]
INSERT 79 7.467452 Epic Movie UNRATED (2007) 720p
INSERT 80 4.616939 National Geographic - The Koran [720p] [Documenter]
INSERT 71 2.871399 Pursuit_Of_Happyness[2006]   720p
INSERT 68 8.971259 Kingdom of Heaven [2005] [720p]
INSERT 82 7.280625 I Am Legend    720p
REMOVE_ID 79
REMOVE_ID 71
REMOVE_ID 49
REMOVE_ID 50
PRINT_IN_ORDER actual/41-test.out
INSERT 43 6.642294 X-Men 3    720p
INSERT 37 7.539727 13.Going.on.30.2004.720p.BluRay.x264-METiS_tehPARADOX.COM
INSERT 16 3.018983 Shinjuku Incident
INSERT 78 9.409205 Yes Man   720p
INSERT 90 4.135293 007 Goldfinger [1964] [BluRay] [720p] [DTS]
INSERT 5 8.017444 The Messenger [720p]
INSERT 65 5.517631 Oceans.Eleven.2001.720p.HDDVD.x264-SiNNERS
INSERT 72 6.813700 War [720p]
INSERT 1 3.259918 Casablana.1942.1080p.BluRay.x264.Codres
INSERT 69 1.022572 James Bond 03 - GoldFinger [1964]
REMOVE_ID 26
REMOVE_ID 9
REMOVE_ID 45
REMOVE_ID 55
REMOVE_ID 80
PRINT_IN_ORDER actual/41-test.out
INSERT 88 9.784565 Music.And.Lyrics.720p.HDDVD.x264-REVEiLLE
INSERT 74 5.614821 Horton Hears a Who! [2008] [720p]
INSERT 29 1.166483 Click   1080p
INSERT 77 2.042661 Pixar.Shorts.2009.720p.BluRay.AC3.x264-UNiT3D
INSERT 86 3.292335 13.Going.on.30.2004.720p.BluRay.x264-METiS_tehPARADOX.COM
INSERT 73 1.472652 The Extreme Movie 720P
INSERT 57 3.346736 Kung Fu Hustle   720p
INSERT 87 9.044111 Resident Evil Series    1080p
INSERT 97 1.425428 Once.Upon.a.Time.in.China.1991.720p.BluRay.x264-BestHD
INSERT 44 8.597600 The.Terminal.2004.720p.HDTV.x264-ESiR
INSERT 83 7.684452 Romeo And Juliet   720p
REMOVE_ID 30
REMOVE_ID 8
REMOVE_ID 82
REMOVE_ID 57
REMOVE_ID 5
PRINT_IN_ORDER actual/41-test.out
INSERT 89 1.781309 Sixth Sense   720p
INSERT 104 9.883037 Duplicity 2009   720p
INSERT 32 7.714015 Four Christmases 720P
INSERT 85 1.356438 Battle for Terra [2007] [1080p]
INSERT 19 4.568424 CJ7   720p
INSERT 94 3.341503 The Bank Job [720p]
INSERT 91 5.622967 Alien vs Predator 1-2  720p
INSERT 107 6.252734 Lord Of The Ring 1 - Fellowship of the Ring (ExtEd) [1080p]
INSERT 98 2.301814 Dark.knight, The.2008.720p.bluray.x264-sinners
INSERT 23 4.069438 Banlieue 13 - Ultimatum [720p]
INSERT 102 8.880765 The Secret of Moonarce [720p]
INSERT 106 4.803281 Click   720p
REMOVE_ID 39
REMOVE_ID 42
REMOVE_ID 37
REMOVE_ID 72
REMOVE_ID 69
REMOVE_ID 90
PRINT_IN_ORDER actual/41-test.out
