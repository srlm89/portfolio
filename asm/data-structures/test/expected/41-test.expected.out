[ 14 6.813328 Pursuit_Of_Happyness[2006]   720p ]
[ 2 9.678840 Eragon [2006] [720p] ][ 7 1.721304 Jumanji [720p] ]
[ 2 9.678840 Eragon [2006] [720p] ][ 3 7.457088 Angels & Demons [2009] [1080p] ][ 17 0.616942 X-Men 2   1080p ][ 28 4.922683 American Pie Book Of Love   720p ]
[ 2 9.678840 Eragon [2006] [720p] ][ 3 7.457088 Angels & Demons [2009] [1080p] ][ 9 5.147998 The Mist   720p ][ 17 0.616942 X-Men 2   1080p ][ 40 3.442556 Braveheart 720p ][ 41 4.001350 Oceans.Eleven.2001.720p.HDDVD.x264-SiNNERS ]
[ 9 5.147998 The Mist   720p ][ 17 0.616942 X-Men 2   1080p ][ 18 4.257201 Demo (Sony, Sharp, THX and more) ][ 25 9.097009 The Lion King [1994] [720p] ][ 27 2.355438 Watchmen [720p] ][ 40 3.442556 Braveheart 720p ][ 41 4.001350 Oceans.Eleven.2001.720p.HDDVD.x264-SiNNERS ][ 46 9.913062 Oceans.Thirteen.2007.720p.HDDVD.x264-SiNNERS ][ 49 9.497091 CJ7   720p ]
[ 6 5.319696 Pixar.Short.Films.Collection.Volume.1.2007.m-HD.x264-TlN50p ][ 9 5.147998 The Mist   720p ][ 17 0.616942 X-Men 2   1080p ][ 21 7.419324 contents.txt ][ 27 2.355438 Watchmen [720p] ][ 34 3.202666 Core   720p ][ 36 0.530856 Rocky 2 [1979] [BluRay] [720p] [AC3] ][ 40 3.442556 Braveheart 720p ][ 46 9.913062 Oceans.Thirteen.2007.720p.HDDVD.x264-SiNNERS ][ 47 0.533557 Deep Sea ][ 48 2.831868 Jeniffer's Body ( 2009)    1080p ][ 49 9.497091 CJ7   720p ]
[ 6 5.319696 Pixar.Short.Films.Collection.Volume.1.2007.m-HD.x264-TlN50p ][ 9 5.147998 The Mist   720p ][ 11 6.491815 Harold & Kumar   720p ][ 17 0.616942 X-Men 2   1080p ][ 21 7.419324 contents.txt ][ 26 3.668788 Alone in the dark   720p ][ 27 2.355438 Watchmen [720p] ][ 35 9.163826 Alvin And The Chipmunks [720p] ][ 40 3.442556 Braveheart 720p ][ 45 6.077245 Blood - The Last Vampire [2009] [720p] ][ 46 9.913062 Oceans.Thirteen.2007.720p.HDDVD.x264-SiNNERS ][ 47 0.533557 Deep Sea ][ 48 2.831868 Jeniffer's Body ( 2009)    1080p ][ 49 9.497091 CJ7   720p ][ 50 5.169793 Braveheart 720p ][ 62 5.557410 Lord Of The Ring 2 - The Two Towers (ExtEd) [1080p] ]
[ 6 5.319696 Pixar.Short.Films.Collection.Volume.1.2007.m-HD.x264-TlN50p ][ 9 5.147998 The Mist   720p ][ 11 6.491815 Harold & Kumar   720p ][ 13 7.774472 Liar Liar    720p ][ 15 7.544752 Final Fantasy The Spirits Within    720p ][ 21 7.419324 contents.txt ][ 22 9.252123 City of Embers    720p ][ 26 3.668788 Alone in the dark   720p ][ 27 2.355438 Watchmen [720p] ][ 30 5.112269 contents.txt ][ 39 0.057899 Lust Caution   720p ][ 40 3.442556 Braveheart 720p ][ 45 6.077245 Blood - The Last Vampire [2009] [720p] ][ 46 9.913062 Oceans.Thirteen.2007.720p.HDDVD.x264-SiNNERS ][ 47 0.533557 Deep Sea ][ 49 9.497091 CJ7   720p ][ 50 5.169793 Braveheart 720p ][ 55 7.714682 Meet The Parents    720p ][ 62 5.557410 Lord Of The Ring 2 - The Two Towers (ExtEd) [1080p] ][ 70 8.577854 James Bond 09 - The Man with the Golden Gun [1974] ]
[ 6 5.319696 Pixar.Short.Films.Collection.Volume.1.2007.m-HD.x264-TlN50p ][ 8 8.228329 Superman Batman Public Enemies 2009 720p ][ 9 5.147998 The Mist   720p ][ 11 6.491815 Harold & Kumar   720p ][ 13 7.774472 Liar Liar    720p ][ 15 7.544752 Final Fantasy The Spirits Within    720p ][ 21 7.419324 contents.txt ][ 22 9.252123 City of Embers    720p ][ 26 3.668788 Alone in the dark   720p ][ 27 2.355438 Watchmen [720p] ][ 30 5.112269 contents.txt ][ 39 0.057899 Lust Caution   720p ][ 40 3.442556 Braveheart 720p ][ 42 3.715998 Ice Age 3 Dawn of the Dinosaurs [2009] [720p] ][ 45 6.077245 Blood - The Last Vampire [2009] [720p] ][ 46 9.913062 Oceans.Thirteen.2007.720p.HDDVD.x264-SiNNERS ][ 47 0.533557 Deep Sea ][ 55 7.714682 Meet The Parents    720p ][ 62 5.557410 Lord Of The Ring 2 - The Two Towers (ExtEd) [1080p] ][ 63 1.182145 The.X.Files.Fight.The.Future.1998.720p.BluRay.x264-SiNNERS ][ 66 2.989426 Etenal Sunshine of the Spotless Mind [2004] [Bluray] [1080p] [DTS] [EN] ][ 68 8.971259 Kingdom of Heaven [2005] [720p] ][ 70 8.577854 James Bond 09 - The Man with the Golden Gun [1974] ][ 80 4.616939 National Geographic - The Koran [720p] [Documenter] ][ 82 7.280625 I Am Legend    720p ]
[ 1 3.259918 Casablana.1942.1080p.BluRay.x264.Codres ][ 5 8.017444 The Messenger [720p] ][ 6 5.319696 Pixar.Short.Films.Collection.Volume.1.2007.m-HD.x264-TlN50p ][ 8 8.228329 Superman Batman Public Enemies 2009 720p ][ 11 6.491815 Harold & Kumar   720p ][ 13 7.774472 Liar Liar    720p ][ 15 7.544752 Final Fantasy The Spirits Within    720p ][ 16 3.018983 Shinjuku Incident ][ 21 7.419324 contents.txt ][ 22 9.252123 City of Embers    720p ][ 27 2.355438 Watchmen [720p] ][ 30 5.112269 contents.txt ][ 37 7.539727 13.Going.on.30.2004.720p.BluRay.x264-METiS_tehPARADOX.COM ][ 39 0.057899 Lust Caution   720p ][ 40 3.442556 Braveheart 720p ][ 42 3.715998 Ice Age 3 Dawn of the Dinosaurs [2009] [720p] ][ 43 6.642294 X-Men 3    720p ][ 46 9.913062 Oceans.Thirteen.2007.720p.HDDVD.x264-SiNNERS ][ 47 0.533557 Deep Sea ][ 62 5.557410 Lord Of The Ring 2 - The Two Towers (ExtEd) [1080p] ][ 63 1.182145 The.X.Files.Fight.The.Future.1998.720p.BluRay.x264-SiNNERS ][ 65 5.517631 Oceans.Eleven.2001.720p.HDDVD.x264-SiNNERS ][ 66 2.989426 Etenal Sunshine of the Spotless Mind [2004] [Bluray] [1080p] [DTS] [EN] ][ 68 8.971259 Kingdom of Heaven [2005] [720p] ][ 69 1.022572 James Bond 03 - GoldFinger [1964] ][ 70 8.577854 James Bond 09 - The Man with the Golden Gun [1974] ][ 72 6.813700 War [720p] ][ 78 9.409205 Yes Man   720p ][ 82 7.280625 I Am Legend    720p ][ 90 4.135293 007 Goldfinger [1964] [BluRay] [720p] [DTS] ]
[ 1 3.259918 Casablana.1942.1080p.BluRay.x264.Codres ][ 6 5.319696 Pixar.Short.Films.Collection.Volume.1.2007.m-HD.x264-TlN50p ][ 11 6.491815 Harold & Kumar   720p ][ 13 7.774472 Liar Liar    720p ][ 15 7.544752 Final Fantasy The Spirits Within    720p ][ 16 3.018983 Shinjuku Incident ][ 21 7.419324 contents.txt ][ 22 9.252123 City of Embers    720p ][ 27 2.355438 Watchmen [720p] ][ 29 1.166483 Click   1080p ][ 37 7.539727 13.Going.on.30.2004.720p.BluRay.x264-METiS_tehPARADOX.COM ][ 39 0.057899 Lust Caution   720p ][ 40 3.442556 Braveheart 720p ][ 42 3.715998 Ice Age 3 Dawn of the Dinosaurs [2009] [720p] ][ 43 6.642294 X-Men 3    720p ][ 44 8.597600 The.Terminal.2004.720p.HDTV.x264-ESiR ][ 46 9.913062 Oceans.Thirteen.2007.720p.HDDVD.x264-SiNNERS ][ 47 0.533557 Deep Sea ][ 62 5.557410 Lord Of The Ring 2 - The Two Towers (ExtEd) [1080p] ][ 63 1.182145 The.X.Files.Fight.The.Future.1998.720p.BluRay.x264-SiNNERS ][ 65 5.517631 Oceans.Eleven.2001.720p.HDDVD.x264-SiNNERS ][ 66 2.989426 Etenal Sunshine of the Spotless Mind [2004] [Bluray] [1080p] [DTS] [EN] ][ 68 8.971259 Kingdom of Heaven [2005] [720p] ][ 69 1.022572 James Bond 03 - GoldFinger [1964] ][ 70 8.577854 James Bond 09 - The Man with the Golden Gun [1974] ][ 72 6.813700 War [720p] ][ 73 1.472652 The Extreme Movie 720P ][ 74 5.614821 Horton Hears a Who! [2008] [720p] ][ 77 2.042661 Pixar.Shorts.2009.720p.BluRay.AC3.x264-UNiT3D ][ 78 9.409205 Yes Man   720p ][ 83 7.684452 Romeo And Juliet   720p ][ 86 3.292335 13.Going.on.30.2004.720p.BluRay.x264-METiS_tehPARADOX.COM ][ 87 9.044111 Resident Evil Series    1080p ][ 88 9.784565 Music.And.Lyrics.720p.HDDVD.x264-REVEiLLE ][ 90 4.135293 007 Goldfinger [1964] [BluRay] [720p] [DTS] ][ 97 1.425428 Once.Upon.a.Time.in.China.1991.720p.BluRay.x264-BestHD ]
[ 1 3.259918 Casablana.1942.1080p.BluRay.x264.Codres ][ 6 5.319696 Pixar.Short.Films.Collection.Volume.1.2007.m-HD.x264-TlN50p ][ 11 6.491815 Harold & Kumar   720p ][ 13 7.774472 Liar Liar    720p ][ 15 7.544752 Final Fantasy The Spirits Within    720p ][ 16 3.018983 Shinjuku Incident ][ 19 4.568424 CJ7   720p ][ 21 7.419324 contents.txt ][ 22 9.252123 City of Embers    720p ][ 23 4.069438 Banlieue 13 - Ultimatum [720p] ][ 27 2.355438 Watchmen [720p] ][ 29 1.166483 Click   1080p ][ 32 7.714015 Four Christmases 720P ][ 40 3.442556 Braveheart 720p ][ 43 6.642294 X-Men 3    720p ][ 44 8.597600 The.Terminal.2004.720p.HDTV.x264-ESiR ][ 46 9.913062 Oceans.Thirteen.2007.720p.HDDVD.x264-SiNNERS ][ 47 0.533557 Deep Sea ][ 62 5.557410 Lord Of The Ring 2 - The Two Towers (ExtEd) [1080p] ][ 63 1.182145 The.X.Files.Fight.The.Future.1998.720p.BluRay.x264-SiNNERS ][ 65 5.517631 Oceans.Eleven.2001.720p.HDDVD.x264-SiNNERS ][ 66 2.989426 Etenal Sunshine of the Spotless Mind [2004] [Bluray] [1080p] [DTS] [EN] ][ 68 8.971259 Kingdom of Heaven [2005] [720p] ][ 70 8.577854 James Bond 09 - The Man with the Golden Gun [1974] ][ 73 1.472652 The Extreme Movie 720P ][ 74 5.614821 Horton Hears a Who! [2008] [720p] ][ 77 2.042661 Pixar.Shorts.2009.720p.BluRay.AC3.x264-UNiT3D ][ 78 9.409205 Yes Man   720p ][ 83 7.684452 Romeo And Juliet   720p ][ 85 1.356438 Battle for Terra [2007] [1080p] ][ 86 3.292335 13.Going.on.30.2004.720p.BluRay.x264-METiS_tehPARADOX.COM ][ 87 9.044111 Resident Evil Series    1080p ][ 88 9.784565 Music.And.Lyrics.720p.HDDVD.x264-REVEiLLE ][ 89 1.781309 Sixth Sense   720p ][ 91 5.622967 Alien vs Predator 1-2  720p ][ 94 3.341503 The Bank Job [720p] ][ 97 1.425428 Once.Upon.a.Time.in.China.1991.720p.BluRay.x264-BestHD ][ 98 2.301814 Dark.knight, The.2008.720p.bluray.x264-sinners ][ 102 8.880765 The Secret of Moonarce [720p] ][ 104 9.883037 Duplicity 2009   720p ][ 106 4.803281 Click   720p ][ 107 6.252734 Lord Of The Ring 1 - Fellowship of the Ring (ExtEd) [1080p] ]
